README.md

```diff
@@ -132,11 +132,11 @@ Prior to submitting a pull request, ensure that your pull request does not conta
 Any sensitive values can be replaced with fake or dummy values for the purposes of submission as long as it is clearly indicated. 
 Also, ensure that the readme.md of the connector is updated with the latest information, or created for new connector submissions. 
 An example of a clear, structured, readme.md can be found in the [Azure Key Vault](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md) connector repository. 
-Include this completed `readme.md` in same connector directory which contains the artifacts. 
+Include this completed `readme.md` in same connector directory which contains the artifacts.Add an appropriate tag using the Labels option indicating which connector type you are submitting. Connector type name should match the folder name you are submitting to: custom-connector, certified-connector, or independent-publisher-connector. 
```

Comment:
I don't think this sentence fits here.

Reply:
@mamurshe , I thought when you open a PR, it would be best to tag it at that time. That is why I added it under "Submitting to the Open Source Repository".  If you think of another place, I am happy to add it there. Also, I see in https://github.com/microsoft/PowerPlatformConnectors#files-to-include I see intro.md to be added which we only suggest the partners to add it in ISV studio.